rr_iarrc/launch/cone_detection_height.launch

```diff
@@ -0,0 +1,28 @@
+<launch>
+    <node name="cone_detection_height" pkg="rr_iarrc" type="cone_detection_height" output="screen">
+
+        <param name="orange_low_H" type="int" value="5" />
+        <param name="orange_high_H" type="int" value="20" />
+        <param name="orange_low_S" type="int" value="150" />
+        <param name="orange_low_V" type="int" value="140" />
+
+        <param name="canny_cut_min_threshold" type="int" value="60" />
```

Comment:
Is this canny's minimum threshold and if so can we get canny's maximum as well as a knob to turn? Leave a note that the max is usually 2-3 times the value. Also, the cut term here is confusing because it usually used for removing things.